.github/README.md

```diff
@@ -161,14 +161,68 @@ Decrypt data with the user's wallet. This is an implementation of the [webcrypto
 
 Requires the `DECRYPT` [permission](#permissions).
 
-### `signature(data, options): Promise<Uint8Array>`
+### ~~`signature(data, options): Promise<Uint8Array>`~~
 
-Get the signature for a data array.
+> ⚠️ **Deprecation warning:** The `signature()` function is deprecated in ArConnect 1.0.0. Read about the alternatives below.
+
+#### Alternatives
+
+There are quite a few cases where you might need to generate a cryptographic signature for a piece of data or message so that you can verify them. The most common ones and their alternatives are the following:
+
+- Generating a signature for a transaction: [`sign()`](#signtransaction-options-promisetransaction)
+- Generating a signature for a bundle data item: [`signDataItem()`](#signdataitemdataitem) or [`dispatch()`](#dispatchtransaction-promisedispatchresult)
+- Signing a message to later validate ownership: [`signMessage()`](#signmessagedata-options-promiseuint8array) combined with [`verifyMessage()`](verify-message.md)
```

Comment:
`verify-message.md` is linking to an empty md file.

Reply:
Should be fixed